common/static/sass/edx-pattern-library-shims/base/_variables.scss

```diff
@@ -178,6 +178,22 @@ $btn-brand-active-background:             $uxpl-blue-base !default;
 $btn-brand-disabled-background:           #f2f3f3 !default;
 $btn-brand-disabled-color:                #676666 !default;
 
+// Upgrade button
+$btn-upgrade-border-color:                $uxpl-green-base !default;
```

Comment:
It looks like @andy-armstrong requested the semantic variables to make this easier to theme. I defer to him on this, but I am still uncomfortable about having "upgrade-button" styles in the general pattern library shim file. Andy, is this the approach you intended?